### PR TITLE
[Native Image] Release py3-native 8.6

### PR DESCRIPTION
**Pull request by samuelFain**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CRTX-105771](https://jira-dc.paloaltonetworks.com/browse/CRTX-105771)

## Description
Release py3-native 8.6

### CODE REVIEW

**Review by JudahSchwartz — APPROVED** [review]
check if we actually need to support a maintenance release. If the feature was never on for 8.4 lets just toss it

File: Tests/docker_native_image_config.json [hunk 1]
```diff
@@ -199,17 +199,17 @@
...
                 "native:candidate"
             ]
         }
     ],
     "flags_versions_mapping":{
         "native:dev":"native:dev",
-        "native:ga":"native:8.4",
-        "native:maintenance":"native:8.3",
+        "native:ga":"native:8.6",
+        "native:maintenance":"native:8.4",
```
Comment (GuyAfik) [inline Tests/docker_native_image_config.json:212]:
```suggestion
```

I think we can remove the `native:maintence` as its not supported in 8.4

File: Tests/docker_native_image_config.json [hunk 2]
```diff
@@ -199,17 +199,17 @@
             "id":"Ping",
             "reason":"see CIAC-8493",
             "ignored_native_images":[
+                "native:8.6",
                 "native:8.4",
```
Comment (GuyAfik) [inline Tests/docker_native_image_config.json:203]:
```suggestion
                "native:8.6",
```

File: Tests/docker_native_image_config.json [hunk 3]
```diff
@@ -181,16 +181,16 @@
...
+                "native:8.6",
+                "native:8.4"
             ]
         },
         {
             "id":"DockerHardeningCheck",
             "reason":"This script is supposed to run inside a container, does not support podman",
             "ignored_native_images":[
+                "native:8.6",
                 "native:8.4",
```
Comment (GuyAfik) [inline Tests/docker_native_image_config.json:193]:
```suggestion
                "native:8.6",
```

File: Tests/docker_native_image_config.json [hunk 4]
```diff
@@ -181,16 +181,16 @@
             "id":"ConvertFile",
             "reason":"Issue: CIAC-7625",
             "ignored_native_images":[
-                "native:8.4",
-                "native:8.3"
+                "native:8.6",
+                "native:8.4"
```
Comment (GuyAfik) [inline Tests/docker_native_image_config.json:185]:
```suggestion
                "native:8.6",
```

File: Tests/docker_native_image_config.json [hunk 5]
```diff
@@ -153,26 +153,26 @@
...
-                "native:8.3",
                 "native:candidate"
             ]
         },
         {
             "id":"RegexExtractAll",
             "reason":"Issue: CIAC-6005",
             "ignored_native_images":[
+                "native:8.6",
                 "native:8.4",
```
Comment (GuyAfik) [inline Tests/docker_native_image_config.json:175]:
```suggestion
                "native:8.6",
```

File: Tests/docker_native_image_config.json [hunk 6]
```diff
@@ -153,26 +153,26 @@
...
-                "native:8.3",
                 "native:candidate"
             ]
         },
         {
             "id":"Intezer v2",
             "reason":"Issue: CIAC-7861",
             "ignored_native_images":[
+                "native:8.6",
                 "native:8.4",
```
Comment (GuyAfik) [inline Tests/docker_native_image_config.json:166]:
```suggestion
                "native:8.6",
```

File: Tests/docker_native_image_config.json [hunk 7]
```diff
@@ -153,26 +153,26 @@
             "id":"Rasterize",
             "reason":"Issue: CRTX-72338 and CIAC-7694",
             "ignored_native_images":[
+                "native:8.6",
                 "native:8.4",
```
Comment (GuyAfik) [inline Tests/docker_native_image_config.json:157]:
```suggestion
                "native:8.6",
```

File: Tests/docker_native_image_config.json [hunk 8]
```diff
@@ -143,8 +143,8 @@
             "id":"FetchIndicatorsFromFile",
             "reason":"issue: CIAC-5243 - ElementTree object (xml) in python3.9 has getiterator attribute while in python 3.10 this attribute does not exist - causing unit tests failures.",
             "ignored_native_images":[
+                "native:8.6",
                 "native:8.4",
```
Comment (GuyAfik) [inline Tests/docker_native_image_config.json:147]:
```suggestion
                "native:8.6",
```

File: Tests/docker_native_image_config.json [hunk 9]
```diff
@@ -133,8 +133,8 @@
             "id":"Symantec MSS",
             "reason":"issue: CIAC-9119 - [Symantec MSS] Module 'OpenSSL.crypto' has deprecated 'load_pkcs12' member",
             "ignored_native_images":[
+                "native:8.6",
                 "native:8.4",
```
Comment (GuyAfik) [inline Tests/docker_native_image_config.json:137]:
```suggestion
                "native:8.6",
```

File: Tests/docker_native_image_config.json [hunk 10]
```diff
@@ -1,6 +1,6 @@
 {
     "native_images":{
-        "native:8.3": {
+        "native:8.4": {
```
Comment (GuyAfik) [inline Tests/docker_native_image_config.json:3]: remove

**Review by GuyAfik — APPROVED** [review]
💯

**Pull request merged**